src/user/user.service.ts

```diff
@@ -147,12 +148,14 @@ export class UserService {
   }
 
   async getMe(req: any) {
-    const user = await this.userRepository.findOne({ where: {id: req.user.id}, include: { all: true }} );
+    const user = await this.userRepository.findOne({
+      attributes: ['id', 'name', 'email', 'roleId', 'pathPhoto', 'teams', 'ban', 'banReason'],
+      where: {id: req.user.id}, include: { all: true }
```

Comment:
👍

Reply:
can also bring `attributes` array to const, since you use it so often. Non-blocking